### PR TITLE
chore: add tests

### PR DESCRIPTION
**Pull request by mhaack**

### CODE REVIEW

**Comment by blefebvre** [comment]
Are there fixtures missing still? When I run `npm test` in the `tests` branch:

```
  1) md2jcr Tests
       convert a simple md:
     Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/simple.xml'
...
  2) md2jcr Tests
       converts headings:
     Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/headings.xml'
...
  3) md2jcr Tests
       converts a document with default-content:
     Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/default-content-only.xml'
```

**Comment by mhaack** [comment]
> Are there fixtures missing still? When I run `npm test` in the `tests` branch:
> 
> ```
>   1) md2jcr Tests
>        convert a simple md:
>      Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/simple.xml'
> ...
>   2) md2jcr Tests
>        converts headings:
>      Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/headings.xml'
> ...
>   3) md2jcr Tests
>        converts a document with default-content:
>      Error: ENOENT: no such file or directory, open '/Users/blefebvr/workspace/github/mhaack/helix-md2jcr/test/fixtures/default-content-only.xml'
> ```

Added them now, there was an entry in git ignore

File: src/mdast2jcr/utils.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export function unwrapImages(mdast) {
     const newChildren = [];
 
     for (const node of section.children) {
-      if (node.type === 'paragraph' || node.type === 'heading') {
+      if (node.type === 'paragraph') {
```
[the remark this answers is not in the document]
Comment (mhaack) [reply src/mdast2jcr/utils.js]: added it back, removed it for some testing

**Comment by mhaack** [comment]
@bhellema @blefebvre there is one issue with the tests: `const nameCounter = {};` is global and not reset in between the tests. Hence there converted xml has different number for each tag. Any idea how this can be reset? I only can this of not making this a global variable?

**Comment by bhellema** [comment]
> @bhellema @blefebvre there is one issue with the tests: `const nameCounter = {};` is global and not reset in between the tests. Hence there converted xml has different number for each tag. Any idea how this can be reset? I only can this of not making this a global variable?

I have changes in my branch iirc that I can look to add a reset.  The current changes support multiple section reseting the counter. I can create a new issue for this.

**Pull request merged**